CHANGES.rst

```diff
@@ -1,6 +1,42 @@
 Release Notes
 =============
 
+2.1.135
+-------
+
+This release brings support for ``pex3 venv {inspect,create}`` for
+working with venvs directly using Pex. Previously, a PEX built with
+``--include-tools`` (or ``--venv``) had the capability of turning itself
+into a venv but the new ``pex3 venv create`` command can do this for any
+PEX file with the addition of a few new features:
+
+#. The venv can now be created directly from requirements producing no
+   intermediate PEX file.
+#. The venv can be created either from a PEX file or a lock file and
+   if requirements are supplied, the resulting venv can be a subset of
+   those.
```

Comment:
I'm slightly confused by this phrasing, is something like this an accurate rephrasing?

```suggestion
#. The venv can be created either from a PEX file or a lock file. A 
   subset either of those can be chosen by also supplying requirements.
```

Reply:
Yup, that's it. Fixed.